vision_datasets/common/dataset_management/dataset_hub.py

```diff
@@ -102,14 +97,14 @@ def create_dataset_manifest(self, container_sas: str, local_dir: str, name: str,
         else:
             dataset_info.index_files = {usage: dataset_info.index_files[usage] for usage in usages if usage in dataset_info.index_files}
 
-        if container_sas and local_dir:
-            downloader = DatasetDownloader(container_sas, self.dataset_registry.get_dataset_info(name, version))
-            downloader_resources_usage = downloader.download(local_dir, usages)
+        if self.container_url and self.local_dir:
+            downloader = DatasetDownloader(self.container_url, self.dataset_registry.get_dataset_info(name, version))
+            downloader_resources_usage = downloader.download(self.local_dir, usages)
```

Comment:
Not related to this PR, but wondering what we should do when we use datasets in multiprocess, e.g. 4gpu training.
Should we just set container_url=null in non-main process?

Reply:
It could be DatasetHub calling the create dataset once and then all GPUs use the dataset?